applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -98,6 +111,22 @@ validate_conference(Context, Id, ?HTTP_PATCH) ->
     patch_conference(Id, Context);
 validate_conference(Context, Id, ?HTTP_DELETE) ->
     load_conference(Id, Context).
+    
+validate(Context, Id, <<"details">>) ->
+    load_conference_details(Context, Id).
+
+validate(Context, Id, Action, ParticipantId) ->
+	AuthDoc = cb_context:auth_doc(Context),
+	ConfDoc = cb_context:doc(load_conference(Id, Context)),
+	ConfOwnerId = wh_json:get_value(<<"owner_id">>, ConfDoc),
+	AuthOwnerId = wh_json:get_value(<<"owner_id">>, AuthDoc),
+	% Abort if the user is not room owner
+	if ConfOwnerId =/= AuthOwnerId ->
+		crossbar_doc:handle_json_success(wh_json:set_value(<<"resp">>, <<"user_not_owner">>, wh_json:new()), Context);
+	true ->
+		Conference = whapps_conference:set_id(Id, whapps_conference:new()),
+		crossbar_doc:handle_json_success(wh_json:set_value(<<"resp">>, perform_conference_action(Conference, Action, ParticipantId), wh_json:new()), Context)
+	end.
```

Comment:
Please use a `case` instead of an `if`.
Also, shorten your lines (by setting variables for example): easier to read, variable names add “documentation”, …